types/jasmine-enzyme/index.d.ts

```diff
@@ -2,7 +2,7 @@
 /// <reference types="react" />
 
 declare function jasmineEnzyme(): void;
-export default jasmineEnzyme;
```

Comment:
jasmine-enzyme was causing this error –
https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/MissingExportEquals.md

Using `export = jasmineEnzyme` fixes it.